openstack/compute/v2/flavors/results.go

```diff
@@ -80,6 +80,12 @@ type Flavor struct {
 	// 65535 characters in length. Only printable characters are allowed.
 	// New in version 2.55
 	Description string `json:"description"`
+
+	// ExtraSpecs is a dictionary of the flavor's extra_specs, and may be
+	// used to avoid extra calls to Get/ListExtraSpecs.  This is only available
+	// in microversion 2.61 and greater, and only if allowed by the compute
+	// service's policy.
+	ExtraSpecs map[string]string `json:"extra_specs"`
```

Comment:
this is already implemented you need to use https://pkg.go.dev/github.com/gophercloud/gophercloud/openstack/compute/v2/flavors#ListExtraSpecs

Reply:
It is already implemented, but you can avoid the N extra round trips and just get the data directly in the /details call.  So above and beyond the obvious performance improvement, and load reduction, it cleans up client code a lot, especially if you have to mess about with go routines to hide the extra calls.

Reply:
To prove my point of how bad it can be here's the naive version:
![extra_specs_naive](https://github.com/gophercloud/gophercloud/assets/5549946/9a0b930e-c4c1-4278-98b6-e54a7f6834cb)

And when you add in a heap of code to parallelize everything:
![extra_specs_parallel](https://github.com/gophercloud/gophercloud/assets/5549946/74cb9000-42a3-4968-b03f-fad27180052f)

Or, what this proposes, is neither of those things and just getting it in the original call.

Reply:
fair point :)